nautobot/docs/user-guide/core-data-model/cloud/cloudservice.md

```diff
@@ -0,0 +1,5 @@
+# Cloud Services
+
+In Nautobot, cloud services enable tracking of resources, applications, and data storage provided to user via remote servers. A cloud service represents a specific instance, within a user's [cloud network](./cloudnetwork.md), of a service defined by its [cloud type](./cloudtype.md). For instance, it could describe a server instance like "c7gn.xlarge" from Amazon EC2 or a Linux virtual machine instance named "M416ms_v2" created using Microsoft Azure Virtual Machine Service within the user's virtual private cloud.
```

Comment:
I would think that `c7gn.xlarge` itself would be a cloud type, rather than a cloud service - an EC2 instance using that cloud type (a specific VM) would be a cloud service.